src/main/java/org/opensearch/flowframework/FlowFrameworkPlugin.java

```diff
@@ -207,4 +213,13 @@ public List<ExecutorBuilder<?>> getExecutorBuilders(Settings settings) {
         );
     }
 
+    @Override
+    public Collection<SystemIndexDescriptor> getSystemIndexDescriptors(Settings settings) {
+        List<SystemIndexDescriptor> systemIndexDescriptors = new ArrayList<>();
+        systemIndexDescriptors.add(new SystemIndexDescriptor(CONFIG_INDEX, "Flow Framework Config index"));
+        systemIndexDescriptors.add(new SystemIndexDescriptor(GLOBAL_CONTEXT_INDEX, "Flow Framework Global Context index"));
+        systemIndexDescriptors.add(new SystemIndexDescriptor(WORKFLOW_STATE_INDEX, "Flow Framework Workflow State index"));
+        return systemIndexDescriptors;
```

Comment:
(Style nit) Already approved, but noting you could use `List.of()` here to keep it similar to all the other extension points, and also can add a one-liner assert to FlowFrameworkPluginTests to get code coverage here.